sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/ConfigurationClientCredentials.java

```diff
@@ -66,7 +68,7 @@
      *
      * @return The base url of the configuration service extracted from connection string provided.
      */
-    String getBaseUri() {
+    public String getBaseUri() {
```

Comment:
getBaseUrl

Reply:
@alzimmermsft  I remembered Alan has an option on it and want to getBaseUri() as the method name.